app/views/registers/show.html.haml

```diff
@@ -54,10 +54,7 @@
               != @register.contextual_data
 
       %h2.heading-medium About this register
-      - if @register.register_phase == 'Backlog'
-        = @register.description
-      - else
-        %p= get_register_definition(@register.id, "register:#{params[:id]}")['text']
+      %p=get_register_definition['text']
```

Comment:
I think this change is wrong

Reply:
This is required because the corresponding method has changed to read from the database record table (derived from the RSF) in: https://github.com/openregister/registers-frontend/blob/delayed-scheduler-backend/app/controllers/registers_controller.rb#L30

Reply:
👍